source/_components/sensor.rflink.markdown

```diff
@@ -58,7 +58,7 @@ devices:
           default: RFLink ID
           type: string
         sensor_type:
-          description: Override automatically detected type of sensor. For list of [values](components/sensor.rflink/#sensors-types) see below.
+          description: Override automatically detected type of sensor. For list of [values](https://www.home-assistant.io/components/sensor.rflink/#sensors-types) see below.
```

Comment:
Actually, we should not use the full url. Just everything after the first `/`. Otherwise it will break the preview sites.

```suggestion
          description: Override automatically detected type of sensor. For list of [values](/components/sensor.rflink/#sensors-types) see below.
```